pages/user/moderation.js

```diff
@@ -2028,7 +2030,35 @@ const UserModerationQueue = ({
                   </a>
                 </span>
                 <span className="col-email text-muted">{profile.content.preferredEmail}</span>
-                <span className="col-created">{formatDateTime(profile.tcdate)}</span>
+                <span className="col-created">
+                  <span
+                    className={`${profile.tmdate < twoWeeksAgo ? 'passed-moderation' : ''}`}
+                  >
+                    {formatDateTime(profile.tmdate, {
```

Comment:
I think it would make more sense for the created date to go first - it will always be the first date chronologically.